tensorflow/python/keras/utils/data_utils.py

```diff
@@ -38,53 +37,11 @@
 import numpy as np
 
 from tensorflow.python.framework import ops
-from six.moves.urllib.request import urlopen
 from tensorflow.python.keras.utils import tf_inspect
 from tensorflow.python.keras.utils.generic_utils import Progbar
 from tensorflow.python.keras.utils.io_utils import path_to_string
 from tensorflow.python.util.tf_export import keras_export
 
-if sys.version_info[0] == 2:
```

Comment:
Please leave this block in place. I've tried to remove it before, and found that it couldn't be done without a deeper change, independently of Python 2 support. The reason is a bit silly, but basically we use the custom `urlretrieve` internally (the `if sys.version_info[0] == 2` gets replaced with `if True`) because it's implemented using `urlopen` and we need to swap out that part to make it work on our systems.

Reply:
Sounds good, I reverted the changes in `data_utils.py`.